src/citrine/resources/data_concepts.py

```diff
@@ -79,56 +85,68 @@ def build(cls, data: dict, session: Session = None):
             An object corresponding to a data concepts resource.
 
         """
+        data_copy = deepcopy(data)
```

Comment:
`deep_copy` is significantly more computationally expensive than a shallow `copy`. Is this required?

Reply:
Looking at this a little deeper, I think you can get away with a single iteration over `data` vaguely like
```python
client_dict = {}
taurus_dict = {}
for k, v in data.items():
    if k in cls._client_keys:
        client_dict[k] = v
    else:
        taurus_dict[k] = v
```

Reply:
It's required because `loads(dumps(foo))` mutates `foo`